src/retrieve-validation/build-form-data.ts

```diff
@@ -0,0 +1,23 @@
+// Imports
+import { Parameters } from '../types/parameters';
+
+// Function that builds multipart/form-data for the specific parameters we pass to the W3C Validator API
+// Based on this standard -> https://datatracker.ietf.org/doc/html/rfc7578
+const buildFormData = (parameters: Extract<Parameters, { text: string }>): string => {
+	const CRLF = '\r\n';
+	const boundary = `--CSSValidatorBoundary`;
+
+	const pieces: string[] = [
+		`Content-Disposition: form-data; name="text"${CRLF}${CRLF}${parameters.text}${CRLF}`,
```

Comment:
Does `parameters.text` need to be encoded in any kind of way? What happens if, for some reason, it contained "Content-Disposition:" or "--CSSValidatorBoundary" in it? Would that break things?

Reply:
Adding onto this, it looks like the way Chrome handles it is by including a randomly-generated portion for the boundary. For example, one boundary for me was: `------WebKitFormBoundaryHAhXdNsJIvpckbGF`.

Reply:
I don't know what Chrome would do if the value happened to somehow guess the random portion correctly and include it, but I assume the odds of that are ridiculously low that it's basically impossible.

Reply:
I don't know about encoding, as the receiving service would have to know to decode the form data in whatever way you encoded it, and that's not something that is part of the `multipart/form-data` standard as far as I know. You're probably right about why Chrome adds the random numbers to the end of its form data boundary, that would explain that behavior, they also include multiple extra dashes that are not required. Similarly the `form-data` NPM package uses a boundary that is just many dashes plus a random number. If we want to do something like that to try to avoid a situation where someone includes `--CSSValidatorBoundary` somewhere in their submitted CSS, I'm not against it. Granted any solution using random numbers is not actually full proof, but it does decrease the likelihood signifcantly.

Reply:
Yeah, let's add more dashes (the same number as Chrome uses) and let's incorporate a random string at the end (also like Chrome).